src/main/java/frc/robot/subsystems/LimeLights.java

```diff
@@ -0,0 +1,119 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.apriltag.AprilTagFields;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Limelight;
+import frc.robot.LimelightHelpers;
+
+import java.util.Optional;
+
+public class LimeLights extends SubsystemBase {
+
+    private static final String LL_Name_Banana = "limelight-banana";
+    private static final String LL_Name_Apple = "limelight-apple";
+    private boolean redAlliance;
+
+    private final Limelight limelight_banana;
```

Comment:
give the limelights real understandable names... like forward/backwards which indicates their position and orientation.